### ISSUE TITLE
Automatically insert overlay in to webpack entry property

### ISSUE THREAD

**Issue by G-Rath** [G-Rath/webpack-serve-overlay#4]
As @shellscape mentioned in [this issue](https://github.com/webpack-contrib/webpack-hot-client/issues/19#issuecomment-367739296); [`webpack-hot-client`](https://github.com/webpack-contrib/webpack-hot-client) does automatic entry injection.

It would be nice if this package did the same, so that users wouldn't have to add the entry manually to their webpack configurations.

**Comment by G-Rath** [comment]
I've had a brief glance already, and looking at it from someone outside the webpack development community, it seems like a pretty complex bit of code.

I'd greatly welcome any input from the surrounding community for this one (and on #1).

**Comment by shellscape** [comment]
Yeah it's not an easy bit of code. Not because it's overly complex, but because there's been a lot of trouble around how automatic injection should be done, due to the many different wants that entries can be setup in configs. 

We might have to look into a way to create "plugin" client scripts for `webpack-hot-client` and have extension scripts bundled with the main hot-client client script.

**Comment by G-Rath** [comment]
In light of the solution for #5, I don't think this feature is 'needed' anymore, at least for now. 

I'm still interested in investigating the works behind AEI, but think that importing the overlay into the bundle is ultimately the better solution, as it sidesteps all the problems associated with AEI.

I'm going to keep this issue open for now, but will likely close it in the future.
